src/screen.rs

```diff
@@ -26,11 +26,21 @@ impl Screen {
     pub fn new() -> Result<Screen> {
         let stdout = MouseTerminal::from(AlternateScreen::from(stdout().into_raw_mode()?));
         Ok(Screen {
-               size: termion::terminal_size()?,
+               size: termion::terminal_size().chain_err(|| ErrorKind::TerminalSize)?,
                stdout: stdout,
            })
     }
 
+    /// Update the terminal size and return `true` if the height changed, and false otherwise.
+    pub fn resize(&mut self) -> Result<Option<(u16, u16)>> {
+        let new_size = termion::terminal_size()?;
+        if self.size.1 == new_size.1 {
```

Comment:
> remove explicit returns?
👍 done

> Also, should we consider implementing Eq for the size tuple so that we don't have to do the dot access?

Yeah but right now I only consider the height, not the width. I don't really know what's the state of line wrapping, so I chose not to handle width changes, I always assume the lines fit.